java/com/google/copybara/git/GitModule.java

```diff
@@ -1199,6 +1200,18 @@ public GitDestination gitHubDestination(
             named = true,
             doc = "Destination reference for the change. By default 'master'",
             defaultValue = "\"master\""),
+        @Param(
+            name = "pr_destination_url",
```

Comment:
Really nice change! I've been wanting to do this for ages :) . Could we use the current user fork by default? In that way the config is stable and doesn't change between who is running it.  Maybe something like: pr_to_fork = True. If there is still the need for doing it for arbitrary repo, we can also have the param you have, and decide based on what fields are set.

Reply:
I could imaging having `push_to_fork = True` that changes the "owner" part of the url to the current user (I think the username is already available, right?). Setting `fork_url` manually is then only required if the fork has a different name, or when pushing on behalf of another user or to an organization.

```starlark
destination = git.github_pr_destination(
    url = "https://github.com/google/copybara.git",
    push_to_fork = True,  # Defaults to False.
    fork_url = "https://github.com/Yannic/copybara.git",  # Defaults to "https://github.com/<current_user>/copybara.git".
)
```

Pointing `url` to the fork and detect the upstream might also work, but, AFAICT, copybara doesn't know the upstream url yet, and the config also would have to change based on who is running it.